### PR TITLE
Update glossary inside DG

### PR DESCRIPTION
**Pull request by ThitipatC**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[github](http://github.com/ananya21)]
 
 - Role: Developer
-- Responsibilities: Data
+- Responsibilities: Datagit fetch origin
```
Comment (modembcc) [inline docs/AboutUs.md]: I think you messed up the console typing.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -592,8 +592,9 @@ _{More to be added}_
 
 ### Glossary
 
-- **Mainstream OS**: Windows, Linux, Unix, MacOS
-- **Private contact detail**: A contact detail that is not meant to be shared with others
+- **Project Manager**:The individual supervising a project whose role may include planning, executing, and closing a project.
+- **Task Status**: A status of a task whether done or not done
+- **Project Status**: A status of a project whether Completed or not Completed
```
Comment (modembcc) [inline docs/DeveloperGuide.md:597]: I like it.

**Review by modembcc — APPROVED** [review]
LGTM

**Review by modembcc — APPROVED** [review]
LGTM

**Review by Lin-Shuang-Shuang — APPROVED** [review]
LGTM

**Pull request merged**